db/migrate/20220520185517_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :name
+      t.text :text
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] The comments table should have  t.references as author_id as a foreign key, and post_id as a foreign key.